docs/design/ocs-to-ocs-api-server.md

```diff
@@ -0,0 +1,162 @@
+# OCS to OCSs : API Server
+
+## Introduction
+This document defines the API server used for communication between OCS provider and OCS consumer clusters.
+
+
+## Tool
+Following tools can be used to create the API server:
+
+- REST (JSON over HTTP)
+- gRPC
+- OpenAPI3
+
+## Tool Comparison
+### gRPC vs REST
+
+- #### Performance:
+    - gRPC is more performant than REST due to smaller payloads (and other reasons).
+    - Current use case does not involve frequent requests between client and server. So performance aspect of the gRPC can be ignored.
+
+- #### Security:
+    - Both provide security with SSL/TLS support.
+    - gRPC endpoint can't be invoked via browser/postman.
+
+We will use gRPC as we are on k8s and gRPC has more advantages compared to REST
```

Comment:
At this stage, the advantages are not clear... Also, I'm not sure why being on k8s justifies using gRPC.

Reply:
Security is a big concern especially when discussing services.
There is no benefit to use REST